docs/docs/reference/configuration.md

```diff
@@ -416,21 +416,53 @@ Defines the stencil URL. Multiple URLs could be given in a comma-separated forma
 * Example value: `http://localhost:8000/testproto.desc`
 * Type: `required`
 
-#### `SCHEMA_REGISTRY_STENCIL_REFRESH_CACHE`
+#### `SCHEMA_REGISTRY_STENCIL_FETCH_TIMEOUT_MS`
 
-Enable/Disable the stencil refresh cache.
+Defines the timeout in milliseconds while fetching the descriptor set from the Stencil server.
 
-* Example value: `false`
+* Example value: `607800`
+* Type: `optional`
+* Default value: `60000`
+
+#### `SCHEMA_REGISTRY_STENCIL_CACHE_AUTO_REFRESH`
+
+Defines whether to enable/disable the auto schema refresh. Please note that auto schema refresh will only work for additions in Enum types in the Proto. It will not fail for other scenarios but it will just ignore any new field additions at the root or nested level, unless the job is restarted.
+
+* Example value: `true`
 * Type: `optional`
 * Default value: `false`
 
-#### `SCHEMA_REGISTRY_STENCIL_TIMEOUT_MS`
+#### `SCHEMA_REGISTRY_STENCIL_REFRESH_STRATEGY`
+
+Defines the schema refresh strategy i.e. `VERSION_BASED_REFRESH` or `LONG_POLLING` when auto schema refresh is enabled. Please note that if the schema refresh strategy is set to `VERSION_BASED_REFRESH` then the `SCHEMA_REGISTRY_STENCIL_URLS` should not be a versioned URL, i.e. it should not have `/versions/xx` at the end. Also note that `VERSION_BASED_REFRESH` strategy will only work if you are using a Stencil server as the schema registry.
+
```

Comment:
I am a bit confused on which one has what behaviour 😅
Let's sync on this ? Wanted to understand a bit more

Reply:
Discussed and resolved.